### PR TITLE
Create Text string animations SVG

### PR DESCRIPTION
**Pull request by Owen-Delisle**

### CODE REVIEW

File: SVG.flood/programs/tags.flp [hunk 1]
```diff
@@ -195,6 +195,42 @@ func patternElementTag(id, x, y, width, height, patternUnits, body)
...
+    width: layer.size.width,
+    height: layer.size.height,
+    fontFamily: layer.font.name,
+    fontSize: layer.fontSize,
+    textAnchor: "middle",
+    style: "inline-size: \(layer.size.width)",
+    fill: layer.foregroundColor.hex,
+    body: body
+  )
+end
```
Comment (jbeenie) [inline SVG.flood/programs/tags.flp]: this is wonderfull to look at

File: SVG.flood/programs/tags.flp [hunk 2]
```diff
@@ -195,6 +195,42 @@ func patternElementTag(id, x, y, width, height, patternUnits, body)
...
+    "fontFamily": fontFamily, 
+    "fontSize": fontSize, 
+    "textAnchor": textAnchor, 
+    "style": style, 
+    "fill": fill, 
+    "transform": transform
+  ]
+
+  return elementTag("text", attributes: attributes, body: body)
+end
```
Comment (jbeenie) [inline SVG.flood/programs/tags.flp]: this is even more wonderful

File: SVG.flood/textBody.svg.flt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <%- if run.attributes["underlineStyle"] != nil && run.attributes["underlineStyle"] != 0 -%>
+    text-decoration="underline"
+    <%- end -%>
+  ><%= run.string %>
+  </tspan>
+  <%- end -%>
+<%- else if layer.string != nil -%>
+  <%= layer.string %>
+<%- end -%>
+<% animations(properties: baseProperties(), layer: layer, timeline: timeline, hrefID: nil, attributeType: nil) -%>
```
Comment (jbeenie) [inline SVG.flood/textBody.svg.flt]: I feel like we can remove line 20? This is already being taken care of in `layer.svg.flt`

Comment (Owen-Delisle) [reply SVG.flood/textBody.svg.flt]: Yep I remove it in a later commit

File: SVG.flood/layer/text.svg.flt [hunk 4]
```diff
@@ -1 +1,14 @@
-<%= textTag(id: id, layer: layer, timeline: timeline) %>
+<% if layer.type.name == "text" %>
```
Comment (jbeenie) [inline SVG.flood/layer/text.svg.flt]: I dont think you need this if statement

File: SVG.flood/textBody.svg.flt [hunk 5]
```diff
@@ -15,6 +15,14 @@
   </tspan>
   <%- end -%>
 <%- else if layer.string != nil -%>
-  <%= layer.string %>
+  <%= keyvalue.value %>
 <%- end -%>
-<% animations(properties: baseProperties(), layer: layer, timeline: timeline, hrefID: nil, attributeType: nil) -%>
\ No newline at end of file
+<% animations(properties: baseProperties(), layer: layer, timeline: timeline, hrefID: nil, attributeType: nil) -%>
```
Comment (jbeenie) [inline SVG.flood/textBody.svg.flt]: why do you need this here?

**Pull request closed without merging**